### DISCUSSION TITLE
Support for V380 Pro

### DISCUSSION THREAD

**Discussion by mascatul** [Arkady23/V380-Web#47] [General]
Hello ,

First of all thank you for the development. 
I was wondering if you could make a package for V380 Pro E32 i attached a picture with software/hardware version.

![image](https://user-images.githubusercontent.com/4599369/178272752-ca075f45-6da1-490f-bd99-a00412fb8443.png)

I found a patch on the internet that is working to enable RTSP , when the camera started before "System Starting" it said "Update Found" or something similar.

I tried to repatch using [bcaller](https://github.com/bcaller) script but it doesn't work , it just says "System Starting" and all that but it doesn't apply the patch so maybe the Hw version is not correct.

Thank you,

**Comment by Arkady23** [comment]
Added a description for installing a web server manually for new cameras.

**Reply by mascatul** [reply]
Thanks il check.
